pyaedt/edb.py

```diff
@@ -1036,9 +1036,8 @@ def create_cutout_on_point_list(
         plane = self.core_primitives.Shape("polygon", points=point_list)
         polygonData = self.core_primitives.shape_to_polygon_data(plane)
         self.core_primitives.create_polygon(
-            plane, list(self.core_stackup.signal_layers.keys())[0], net_name="DUMMY_CUTOUT"
```

Comment:
you cannot rename DUMMY_CUTOUT. the DUMMY word into the net name has to be preserved otherwise the solver will mesh it. We don't want to have it meshed since it is an added polygon.

Reply:
@maxcapodi78 Sure I will revert it.

Reply:
Done